Tests/SwerverTests/ActionSpec/NotAllowedActionSpec.swift

```diff
@@ -0,0 +1,23 @@
+import Quick
+import Nimble
+import Swerver
+
+class NotAllowedActionSpec: QuickSpec {
+    
+    override func spec() {
+        describe("A Method Not Allowed action") {
+            let responseBuilder = ResponseBuilder()
+            let notAllowedAction = NotAllowedAction()
+            
+            it("can execute to response builder to build a response for a successful OPTIONS request") {
```

Comment:
Did you copy/paste "execute"? I am not sure the test description makes sense.

Reply:
search and replace didn't work as intended. will fix that

Reply:
fixed